en/drafts/originals/creating-guis-in-python-for-digital-humanities-projects.md

```diff
@@ -25,26 +25,13 @@ doi: XX.XXXXX/phen0000
 
```

Comment:
@yann-ryan, please use the numerical codes 1 (Beginner), 2 (Intermediate), or 3 (Advanced) to indicate difficulty.
For example, Intermediate = `difficulty: 2`

Choose one activity: acquiring, transforming, analyzing, presenting, sustaining

Choose one (or more) topics from those [listed here](https://github.com/programminghistorian/jekyll/blob/gh-pages/_data/topics.yml). The structure follows this example: `topics: [python, data-manipulation]`. If none of the topics in the list feel right, we can chat with Alex and create a new one.

Work with Christopher to write a short abstract of the lesson to form the summary available beneath the lesson title in our [Lesson Index](https://programminghistorian.org/en/lessons/). It will also be included within the lesson’s header.

Have you selected an image to represent this lesson? I can send you guidelines for what is needed. We'll want to slot in the `avatar_alt:` to accompany it here.

Reply:
We've completed these, and have an avatar ready to go. Can you let me know where to upload it and how to correctly link it to the lesson?

Reply:
Dear @yann-ryan. I have emailed you a step-by-step for how to create and upload an avatar image. 

You can upload directly to ph-submissions gh-pages – you don't need to work in this branch.